providers/google.go

```diff
@@ -39,19 +39,18 @@ import (
 	httphelper "github.com/zitadel/oidc/v3/pkg/http"
 )
 
-var (
-	issuedAtOffset = 1 * time.Minute
-)
-
 const googleIssuer = "https://accounts.google.com"
 
 type GoogleOptions struct {
-	ClientID     string
-	ClientSecret string
-	Issuer       string // This should almost always be "https://accounts.google.com"
-	Scopes       []string
-	RedirectURIs []string
-	GQSign       bool
+	ClientID       string
+	ClientSecret   string
+	Issuer         string // This should almost always be "https://accounts.google.com"
+	Scopes         []string
+	RedirectURIs   []string
+	GQSign         bool
+	OpenBrowser    bool
```

Comment:
Can you add a comment here explaining some of these new struct variables? For instance no one should set OpenBrowser = false outside of integration tests

Reply:
Comments added here: 53cbe3e